package.json

```diff
@@ -30,19 +29,17 @@
   "scripts": {
     "presystem-test": "npm run compile",
     "system-test": "mocha build/system-test --timeout 600000",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test": "c8 mocha build/test/*{,/*}.js",
-    "lint": "eslint '**/*.js' && gts check",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "test": "mocha build/test --timeout 600000",
```

Comment:
This should have `c8 mocha` to ensure code coverage reporting.  Also, I don't think we want that high of a timeout for our unit tests, do we?

Reply:
Yeah, I thought this was 60000 previously.

Reply:
I checked, it was totally 600000. Maybe not part of this then, but it does seem like something we should change.

Reply:
make change to `60000` instead of `600000`